### PR TITLE
Refactor protein alignment example

### PR DESCRIPTION
**Pull request by fwmeng88**
- [x] Refactorization
- [x] Add binder bage to link the Jupyter notebook

### CODE REVIEW

**Review by FarnazH — CHANGES_REQUESTED** [review]
@fwmeng88 Is there any way to just render the Jupyter notebook here? I don't see any reason to repeat the same content/code snippet, and this also makes it hard to update examples/tutorials.

I know this is possible for an example python script, so I believe one can do the same for the notebook.

File: docs/usr_doc_chem_align.rst [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    :
+    : --
+
+
+Chemical Structure Alignment
+============================
+
+Molecular alignment is a fundamental problem in cheminformatics and can be used for structure
+determination, similarity based searching, and ligand-based drug design et al. This problem can be
+solved by orthogonal Procrustes when given two matrices representing three-dimensional
```
Comment (Ali-Tehrani) [inline docs/usr_doc_chem_align.rst:30]: "... solved as a orthogonal Procrustes problem ..."

Comment (fwmeng88) [reply docs/usr_doc_chem_align.rst:30]: Thanks for this. It's resolved in 5239ae4.

**Comment by fwmeng88** [comment]
I am going to close this as we are using `nbsphinx` to convert notebooks for the website (#106).

**Pull request closed without merging**